src/components/cart/cart-view/cart-view.tsx

```diff
@@ -75,6 +77,12 @@ export const CartView = ({
                         >
                             Checkout
                         </button>
+                        <button
```

Comment:
Make it a link

Reply:
I would stay with button, because I need to close right panel. 
```
  const onViewCart = () => {
    setIsOpen(false);
    navigate(ROUTES.cart.to());
  };
```
Look not nice:
```
<Link to={ROUTES.cart.to()}
      onClick={onViewCart} 
>
```

Reply:
I don't see a problem with `onClick`. Looks ok to me.